app/controllers/internal/profile_fields_controller.rb

```diff
@@ -1,32 +1,50 @@
 module Internal
   class ProfileFieldsController < Internal::ApplicationController
+    ALLOWED_PARAMS = %i[
+      input_type label active placeholder_text description
+    ].freeze
     layout "internal"
 
     def index
       @profile_fields = ProfileField.all
     end
 
     def update
-      @profile_fields = ProfileField.find(params[:id])
-      @profile_fields.update!(profile_field_params)
+      profile_field = ProfileField.find(params[:id])
+      if profile_field.update(profile_field_params)
+        flash[:success] = "Profile Field updated"
+      else
+        flash[:error] = "Profile Field error: #{profile_field.errors_as_sentence}"
```

Comment:
It's probably redudant to tell the user "Profile Field" as a prefix of the error message. That's the only thing they can edit in this page